bm-persona/Dining/DiningDataSource/DiningItem.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  DiningItem.swift
+//  bm-persona
+//
+//  Created by Kevin Hu on 3/14/20.
+//  Copyright © 2020 RJ Pimentel. All rights reserved.
+//
+
+import Foundation
+
+/** DiningMenu is a typealias for an array of DiningItems. */
+typealias DiningMenu = Array<DiningItem>
+
+/** Object representing a single dish/item. */
+class DiningItem {
+    
+    let name: String
+    
+    // Additional information regarding food such as allergens or alternative options
+    let restrictions: [String]
```

Comment:
do we want to save restrictions as strings? zeplin shows filters for gluten-free, vegan, etc. so we might want to use an enum for all the categories a dining item fits

Reply:
It's stored in Firebase as an array of strings so likely just placed into this.

Reply:
Yea. I just wanted to make sure we could still display new restrictions that the app isn't explicitly programmed to handle. I think the enum is a good idea, though, so I pushed a commit that adds a KnownRestriction enum (so we can display things like icons for the ones we know) and a DiningRestriction class that will allow us to handle both known and unknown restrictions.